src/utils.py

```diff
@@ -99,7 +104,7 @@ def rasterize(x, resolution=64, sigma=0.01, device='cuda'):
     return raster
 
 
-def save_images(images, size, path):
+def save_images_simple(images, size, path):
```

Comment:
If I understand correctly, `save_images_simple` and `save_images_real` correspond to the simple and real raster, respectively? The only some minor differences between these two functions, so it might be better to have one `save_images` function that has a one or two if statements inside.

Reply:
Eh, whatever.